lib/core/modules/coderunner/codeRunner.js

```diff
@@ -93,6 +93,11 @@ class CodeRunner {
       const value = await result;
       cb(null, value);
     } catch (error) {
+      // Improve error message when there's no connection to node
+      if (error.message.indexOf('Invalid JSON RPC response') !== -1) {
```

Comment:
I'm not sure how much control we have over the `error` coming in from `result`, but maybe we want to at least put this string `'Invalid JSON RPC response'` in constant?

Reply:
This error specifically comes from here: https://github.com/ethereum/web3.js/blob/develop/lib/web3/errors.js#L37, unfortunately there were no importable constant or cleaner way to detect that it's in fact that error.

Extracted this string into constant.

Reply:
Since we're in catch block `error` is always an instance of `Error`, appending `message` seems like a good approach to me, as it doesn't alter original message (in case it contains something useful) and preserves original stack trace (which is impossible if we create `new Error`).

Reply:
@evgeniuz thanks for taking the time to clarify things here. 

Yes, this all make sense. I was mainly pointing at the string check and the we might wanna put that in a constant. However maybe it's also already clear enough.

In general I personally just try to avoid introducing "Magic" strings.

Hope this makes sense!

Reply:
Magic string is now in constant in same file. It's also easier to see that way that error comes from web3.